openfisca_france/model/prestations/aide_permis_probtp.py

```diff
@@ -0,0 +1,35 @@
+from openfisca_france.model.base import Variable, Individu, MONTH
+from numpy import select
+
+
+class aide_permis_probtp(Variable):
+    value_type = float
+    entity = Individu
+    label = "Montant de l'aide au permis B PROBTP"
```

Comment:
Ajouter un espace comme cela est fait sur le site PRO BTP ?
```suggestion
    label = "Montant de l'aide au permis B PRO BTP"
```
Et c'est plus facile à lire ^^

Reply:
> On pourrait également renommer la variable en `aide_permis_pro_btp` (et uniformiser sur les path des paramètres en fonction).

Reply:
Oui en effet c'est mieux d'uniformiser, je vais également corriger ça !